packages/kotti-ui/source/kotti-breadcrumb/KtBreadcrumb.vue

```diff
@@ -72,17 +75,25 @@ export default defineComponent<KottiBreadcrumb.PropsInternal>({
 	}
 
 	&__text {
-		&--is-completed {
-			font-weight: 600;
+		font-weight: 400;
+		color: var(--text-01);
+		&:hover {
+			color: var(--primary-70);
+			cursor: pointer;
+		}
+		&--is-completed,
+		&--is-active {
 			color: var(--breadcrumb-color-completed);
-
-			&:hover {
-				cursor: pointer;
-			}
 		}
 		&--is-active {
-			font-weight: 600;
-			color: var(--breadcrumb-color-active);
+			font-weight: 700;
+		}
+		&--is-disabled {
+			color: var(--text-05);
+			&:hover {
+				color: var(--text-05);
```

Comment:
i think you needed this because of specifity conflict.

do you know which?

Reply:
actually i don't see any thing that can conflict

do you need the duplication?